demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/MainActivity.kt

```diff
@@ -66,7 +66,7 @@ class MainActivity : ComponentActivity() {
                             Button({ content = 7 }) { Text("7") }
                         }
                         when (content) {
-                            1 -> DatingCards()
+                            1 -> Puzzle15()
```

Comment:
We still need the `DatingCards`, can you add this as a new item to the bottom of the list?